core/sdk-core/src/main/java/software/amazon/awssdk/core/async/AsyncRequestBody.java

```diff
@@ -160,6 +160,37 @@ static AsyncRequestBody fromByteBuffer(ByteBuffer byteBuffer) {
         return fromBytes(BinaryUtils.copyAllBytesFrom(byteBuffer));
     }
 
+    /**
+     * Creates a {@link BlockingOutputStreamAsyncRequestBody} to use for writing to the downstream service as if it's an output
+     * stream.
+     *
+     * <p><b>Example Usage</b>
+     *
+     * <pre>
```

Comment:
Nit, can we use snippet? :) 
```
{@snippet :
  
}
```

Reply:
Have you merged the PR that enables snippets yet? I searched around and we weren't using snippets anywhere else in master.

Reply:
Inline snippets should be supported now. I haven't merged the change to enable external snippets.